src/org/loklak/api/search/MeetupsCrawlerService.java

```diff
@@ -63,7 +63,7 @@ public JSONObject serviceImpl(Query call, HttpServletResponse response, Authoriz
 	}
 
 	public static SusiThought crawlMeetups(String url) {
-		
```

Comment:
The spacing is not replaced with a newline, I have just removed the unnecessary trailing spaces.